src/client/styles/create.css

```diff
@@ -26,3 +26,7 @@ table {
 .spectator-row {
   border-top: 5px double #eee;
 }
+
+.text-input-wide {
+  width: 150%;
```

Comment:
setting the width to > than 100 % seems a bit weird, since it means that the element would be scaled to be 1.5 times the size of its parent. What was the intention of this styling?

Reply:
You are right. Can't remember, what the reason was. Changed it to 100%.